src/patternfly/components/Page/page.scss

```diff
@@ -285,6 +295,40 @@
       display: block;
       visibility: visible;
     }
+
+    > .pf-c-button {
```

Comment:
This works, but where it will break down is if buttons are used in a `.pf-c-page__header-tools-group` element without `.pf-m-icons` - the modifier is only used to hide the group on mobile. So if someone defined a `.pf-c-page__header-tools-group` with buttons inside of it that they want to persist on mobile (by omitting `.pf-m-icons`), they wouldn't get the focus styles. Seems like it would be safe to to move this so it applies to any `> .pf-c-button` in a `.pf-c-page__header-tools-group`.

Reply:
agreed, moved it outside of `.pf-m-icons`